test/passes/br_if_to_try.bin.txt

```diff
@@ -0,0 +1,19 @@
+(module
```

Comment:
Nit: This contains `br`, not `br_if`, so maybe the file name is misleading..?

Reply:
Also if this does not need any "passes", maybe we can put the wasm file in test/ ? (I'm not really sure what the convention is, I guess you know better though)

Reply:
Good points, yes, this is better in `test/`. Will move and simplify as you suggest.